eng/common/tools.ps1

```diff
@@ -285,6 +285,25 @@ function InitializeXCopyMSBuild([string]$packageVersion, [bool]$install) {
   return Join-Path $packageDir "tools"
 }
 
+function EnablePreviewSdks() {
```

Comment:
We can't modify anything under `eng/common/`; that's all a mirror of arcade scripts.  Try to modify `eng/Build.ps1`, instead.  If you must modify something under `eng/common`, you'll have to submit a PR to [dotnet/arcade](https://github.com/dotnet/arcade) and wait for the change to flow through.

Reply:
Makes sense. I moved it out, it was no problem. I didn't know it was a mirror.

Reply:
@brettfo would it make sense to have a this.is.mirror.txt in this folder or maybe a readme.md in folder above stating the matter?

Reply:
@brettfo I moved it out of tools.ps1 and nothing in common should be modified. This ok with you?